installer/Migration/V5_1_0/lang-string/recruitment.yaml

```diff
@@ -20,9 +20,6 @@ langStrings:
   -
     value: 'Performed Action'
     unitId: performed_action
-  -
-    value: 'Performed By'
```

Comment:
@SandeepaOHRM I removed this duplicate string since it is already there in general.yaml in 5.0

recruitment.performed_by doesn't seem to be used but please check if possible

Reply:
Seems like not in use. You can remove